specs/precompile/08ecPairing.md

```diff
@@ -0,0 +1,52 @@
+# ecPairing precompile
+
+## Procedure
+
+The `ecPairing` precompile computes the bilinear map between two given points in the groups G1 and G2, respectively, over the alt_bn128 curve.
+
+### Circuit behavior
+
+The input is arbitrarily many pairs of elliptic curve points. Each pair is given as a six 32-bytes values and is constructed as follows
+
+```
+input[0; 31] (32 bytes): x1
+input[32; 63] (32 bytes): y1
+input[64; 95] (32 bytes): x2
+input[96; 127] (32 bytes): y2
+input[128; 159] (32 bytes): x3 (result)
+input[160; 192] (32 bytes): y3 (result)
```

Comment:
```suggestion
input[160; 191] (32 bytes): y3 (result)
```

Reply:
fixed in [50a8a56](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/504/commits/50a8a56ef4730ddb4884a637d3c5b00978fa14a4)